locales/en/message-browser.json

```diff
@@ -0,0 +1,44 @@
+{
+  "field": {
+    "epoch": "Epoch timestamp",
+    "headers": "Headers",
+    "key": "Key",
+    "offset": "Offset",
+    "partition": "Partition",
+    "size": "Size",
+    "timestamp": "Timestamp",
+    "value": "Value"
+  },
+  "filter": {
+    "epoch": "Epoch timestamp",
+    "epoch_aria_label": "Select epoch timestamp",
+    "epoch_placeholder": "Select epoch timestamp",
+    "latest": "Select latest",
+    "offset": "$t(field.offset)",
+    "offset_aria_label": "Select offset",
+    "offset_placeholder": "Select offset",
```

Comment:
One question I had for @simrandhaliw is whether using "Select ..." in the text inputs is appropriate. This is recommended for selecting an option in a menu, but I don't see an equivalent recommendation when the value is manually specified by the user. I'm wondering if instead of "Select epoch timestamp" should be replaced with "Specify epoch timestamp".

Same comment applies here for the offset placeholder.

Reply:
Yes, I agree. Using 'Select' isn't proper in this case. I asked for suggestions in the content-help slack channel, and the two suggestions I received were to use 'Specify' or 'Enter' instead of 'Select'.